scripts/openmc-track-to-vtk

```diff
@@ -41,25 +39,22 @@ def main():
     point_offset = 0
     for fname in args.input:
         # Write coordinate values to points array.
-        track = h5py.File(fname)
-        n_particles = track.attrs['n_particles']
-        n_coords = track.attrs['n_coords']
-        coords = []
-        for i in range(n_particles):
-            coords.append(track['coordinates_' + str(i + 1)][()])
-            for j in range(n_coords[i]):
-                points.InsertNextPoint(coords[i][j,:])
-
-        for i in range(n_particles):
-            # Create VTK line and assign points to line.
-            line = vtk.vtkPolyLine()
-            line.GetPointIds().SetNumberOfIds(n_coords[i])
-            for j in range(n_coords[i]):
-                line.GetPointIds().SetId(j, point_offset + j)
-
-            # Add line to cell array
-            cells.InsertNextCell(line)
-            point_offset += n_coords[i]
+        track_file = openmc.Tracks(fname)
+        for track in track_file:
+            for particle in track.particles:
```

Comment:
```suggestion
            for particle in track.particle_tracks:
```

Reply:
Beyond the scope of this PR but perhaps one day we should make this export to vtk capability part of the track class, just like the plot method.